README.md

```diff
@@ -164,9 +164,9 @@ List<Payment> payments = razorpayClient.Orders.fetchPayments("order_id");
 You can use the Utils class to verify the signature received in response to a payment made using Orders API
 ```java
 JSONObject paymentResponse = new JSONObject();
-options.put("razorpay_order_id", "<order_id>");
-options.put("razorpay_payment_id", "<payment_id>");
-options.put("razorpay_signature", "<signature>");
+paymentResponse.put("razorpay_order_id", "<order_id>");
```

Comment:
Thanks @Archit-kumar for pointing this and giving a fix.

To be consistent with other code snippets in readme here, suggest to do s/paymentResponse/options instead.